torch/_dynamo/variables/nn_module.py

```diff
@@ -222,6 +222,9 @@ def _custom_getattr_fallback(self, base, tx, name, options):
         if not isinstance(getattr_fn, types.FunctionType):
             unimplemented("torch.nn.Module with a non-function custom __getattr__")
 
+        if getattr(base, "_is_fsdp_managed_module", False):
```

Comment:
Note to self: on current main, we should be tracing through all `nn.Module`s (`TORCHDYNAMO_INLINE_INBUILT_NN_MODULES=1` will be the default very soon). We should enable that envvar and remove all changes in this file (and related files), and see what fails.